packages/babel-helper-replace-supers/package.json

```diff
@@ -14,6 +14,7 @@
   },
   "main": "./lib/index.js",
   "dependencies": {
+    "@babel/core": "workspace:^",
```

Comment:
This should need to be a peer dependency, though I'm not sure, see what other people think.

Reply:
It seems that other packages use this:
```json
"peerDependencies": {
  "@babel/core": "^7.0.0-0"
},
"devDependencies": {
  "@babel/core": "workspace:^"
},
```